examples/haas.cfg

```diff
@@ -114,3 +114,8 @@ haas.ext.auth.database =
 # are required.
 vlans = 100-200
 
+[haas.ext.switches.dell]
+# This section can be used when dell switch is in use.
+# It enables saving the switch configuration to memory.
```

Comment:
This is a little unclear. When someone says "memory" I immediately think "RAM" which is not what we're talking about here. How about `Make modifications to the dell switches' configuration persistent:`?